rabbitmq-operator/src/main/java/com/indeed/operators/rabbitmq/reconciliation/RabbitMQClusterReconciler.java

```diff
@@ -83,8 +83,8 @@ public void reconcile(final Reconciliation reconciliation) throws InterruptedExc
                 deleteDanglingPvcs(resource, currentReplicaCount);
             }
 
-            shovelReconciler.reconcile(cluster);
             usersReconciler.reconcile(cluster);
+            shovelReconciler.reconcile(cluster);
```

Comment:
Is there a reason for these to have swapped positions?  If there's a necessary order, that'd make a great comment.

Reply:
added